spec/mailers/generic_mailer_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GenericMailer, type: :mailer do
+  DEFAULT_FROM =
+    'postmaster@sandbox8f696e611a6e4906a977c007f8971322.mailgun.org'
+  describe '#drop_duty' do
+    let(:mail) do
```

Comment:
Nice specs :) might want to have some blank lines yeah? So that it doesn't look too cluttered.